x/oracle/types/keys.go

```diff
@@ -18,5 +20,26 @@ const (
 )
 
 var (
-// KeyPrefixOracles defines key to store oracle
+	// KeyPrefixOracle defines key to store oracle
+	KeyPrefixOracle                 = []byte{0x01}
+	KeyPrefixOracleRegistration     = []byte{0x02}
+	KeyPrefixOracleRegistrationVote = []byte{0x03}
+
+	KeyIndexSeparator = []byte{0xFF}
 )
+
+func GetKeyPrefixOracle(address string) []byte {
+	return append(KeyPrefixOracle, []byte(address)...)
```

Comment:
There is no issue to be modified but I'd like to say and discuss with you regarding how to use the address for key of KV store.

There are two simple ways for converting string type of address to byte array:
1. direct converting (as you did in this PR)
2. remove prefix(panacea1) and convert to byte (using `sdk.AccAddressFromBech32`, which returns sdk.AccAddr)

I think both are fine but it would be good if we use the consistent way for future. wdyt?

Reply:
Yes, I was thinking about this too.
Thinking about it now, the second method you mentioned seems better. (using `sdk.Address`)
Whay is your opinion?

Reply:
Please rename the function to `GetOracleKey`. Also, for other functions below.

Better go with `sdk.AccAddr`.
https://github.com/cosmos/cosmos-sdk/blob/09553f10417d8c78d8bb76ef2564e2b1967d9ac1/x/staking/types/keys.go#L59

Reply:
Yes. I modified it to use `sdk.AccAddr`.
And changed key name.